userguide/jails.rst

```diff
@@ -1276,45 +1234,40 @@ is displayed:
 
 
 Use the arrow keys to select an option and press :kbd:`spacebar`
-to toggle the value. When all the values are as desired, press
-:kbd:`Enter`.  The port will begin to compile and install.
+to toggle the value. Press :kbd:`Enter` when satisfied with the jail
+options. The port begin to compile and install.
 
-.. note:: The configuration screen will not be shown again, even
-   if the build is stopped and restarted. It can be redisplayed
-   by typing :command:`make config`.  Change the settings, then
-   rebuild with :command:`make clean install clean`.
+.. note:: The configuration screen is not shown again. Redisplay it by
+   typing :command:`make config`. Change the settings,
+   then rebuild with :command:`make clean install clean`.
 
-Many ports depend on other ports. Those other ports can also have
-configuration screens that will be shown before compiling begins. It
-is a good idea to keep an eye on the compile until it finishes and the
+Many ports depend on other ports. Those other ports also have
+configuration screens that are shown before compiling begins. It
+is a good idea to watch the compile until it finishes and the
 command prompt returns.
 
-When the port is installed, it is registered in the same package
-database that manages packages. The same :command:`pkg info` command
-can be used to determine what was installed, as described in the
-previous section.
-
+The port is registered in the same package database that manages
+packages when it is installed. The :command:`pkg info`
```

Comment:
packages. :command:`pkg info` can be used to determine
which ports were installed.

(Although it's not that easy, really.)